src-frontend/app/components/choice-list.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    targetObject: Em.computed.alias('parentView'),
+    didInsertElement: function() {
+        var choiceListComponent = this;
+        this.$("ul.choices").sortable({
+            handle: 'span.drag-handle',
+            revert: true,
+            scroll: true,
+            update: function(event, ui) {
+                var choices = $(this).sortable('toArray', { attribute: 'choice' });
+
+                $(this).sortable('cancel');
```

Comment:
Yeah, this is what Fasih and I spent 4 hours figuring out. Otherwise jquery-sortable overrides the ordering and messes the model up.